packages/graphql-yoga/__tests__/http-extensions.spec.ts

```diff
@@ -39,120 +32,4 @@ describe('GraphQLError.extensions.http', () => {
     expect(response.status).toBe(401)
     expect(response.headers.get('www-authenticate')).toBe('Bearer')
   })
-
-  it('picks the highest status code and headers for GraphQLErrors thrown within multiple resolvers', async () => {
```

Comment:
We have this because we want to give the user to control the response parameters. Also we benefit from this in our plugins in the execution level.

Reply:
Throwing errors from field resolvers will nullify only that field, for example:
```gql
query { a }
```
and resolver for `a` errors out, the response will be:
```js
{ data: { a: null }, errors: [{ message: 'Woah!' }] }
```

As per the spec, if the `data` field is not-null, the response code must be 200. (For [accepting application/json](https://graphql.github.io/graphql-over-http/draft/#sel-EANNLDJAADLCAnwD) and for [accepting application/graphql+json](https://graphql.github.io/graphql-over-http/draft/#sel-EANNNDFAADHCA9vK))